fabrics.c

```diff
@@ -47,6 +47,7 @@
 
 #define PATH_NVMF_DISC		SYSCONFDIR "/nvme/discovery.conf"
 #define PATH_NVMF_CONFIG	SYSCONFDIR "/nvme/config.json"
+#define PATH_NVMF_RUNDIR	RUNDIR "/nvme"
```

Comment:
Should `PATH_NVMF_RUNDIR` be set to `/run/nvme`. I believe that since FHS 3.0 the use of `/run` is encouraged for volatile run time data (but I may be wrong...).

Ref: https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard

Reply:
`RUNDIR` is set to `run` as default, thus it will looking under `${PREFIX}/run/nvme`

Reply:
> BTW, I using `/etc/tmpfiles.d/nvme-cli.conf ` to create the directory:
> 
> `d /run/nvme 0755 - - -`

Good idea!

Reply:
I assume that each application will be in charge of writing their configuration to this directory using a unique file name (e.g. `/run/nvme/nvme-stas.json`). This ensures that there won't be collisions between two apps trying to write to the same file at the same time.